modules/jaggery-apps/portal/gadgets/user_profile/controllers/my-profile/totp-init.jag

```diff
@@ -0,0 +1,52 @@
+<%
+/*
+ * Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+include("../wsUtil.jag");
+var util = require("../../util/utility.jag");
+var log = new Log();
+var serverUrl = util.getServerUrl();
+serverUrl = serverUrl + "/TOTPAdminService";
+
+var ws = require('ws');
+var webService = new ws.WSRequest();
+
+function init(){
+
+    var cookie = session.get('auth-cookie');
+    if(cookie == null){
+        var cookie = request.getParameter("cookie");
+        session.put('auth-cookie', cookie);
+    }
+    var userName = request.getParameter("user");
+    try{
+        if (log.isDebugEnabled()) {
+            log.debug("Calling the web service url "+serverUrl);

```

Comment:
log.debug("Calling the web service url: " + serverUrl);

Format code like above